src/schedule/components/ScheduleWrapper.tsx

```diff
@@ -0,0 +1,20 @@
+import React from "react";
+import ScheduleHeader from "./ScheduleHeader";
+
+interface ScheduleWrapperProps {
+  children: React.ReactNode;
```

Comment:
```suggestion
interface ScheduleWrapperProps extends PropsWithChildren {}
```

[Lv1] PropsWithChildren 을 사용하면, 이후 추가될 props만 넣을수있도록 인터페이스 구성할수있습니다!